frontend/src/app/pages/exercises/exercise/shared/simulation/signaller-modal/details-modal/signaller-modal-start-transfer-of-category-modal/signaller-modal-start-transfer-of-category-modal.component.ts

```diff
@@ -0,0 +1,163 @@
+import type { OnDestroy, OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import type {
+    PatientStatus,
+    PatientStatusForTransport,
+} from 'digital-fuesim-manv-shared';
+import type { UUID } from 'digital-fuesim-manv-shared';
+import { Subject } from 'rxjs';
+import { ExerciseService } from 'src/app/core/exercise.service';
+import type { HotkeyLayer } from 'src/app/shared/services/hotkeys.service';
+import { HotkeysService } from 'src/app/shared/services/hotkeys.service';
+import type { AppState } from 'src/app/state/app.state';
+import { createSelectBehaviorStatesByType } from 'src/app/state/application/selectors/exercise.selectors';
+import { selectStateSnapshot } from 'src/app/state/get-state-snapshot';
+import { MessageService } from 'src/app/core/messages/message.service';
+import { SignallerModalDetailsService } from '../signaller-modal-details.service';
+
+@Component({
+    selector: 'app-signaller-modal-start-transfer-of-category-modal',
+    templateUrl:
+        './signaller-modal-start-transfer-of-category-modal.component.html',
+    styleUrls: [
+        './signaller-modal-start-transfer-of-category-modal.component.scss',
+    ],
+    standalone: false,
+})
+export class SignallerModalStartTransferOfCategoryModalComponent
+    implements OnInit, OnDestroy
+{
+    @Input() simulatedRegionId!: UUID;
+
+    private hotkeyLayer!: HotkeyLayer;
+    private readonly destroy$ = new Subject<void>();
+
+    transportStarted = false;
+    maximumStatus: PatientStatusForTransport = 'red';
+
+    loadingCount = 0;
+
+    allowedStatuses = ['red', 'yellow', 'green'] as const;
```

Comment:
```suggestion
    allowedStatuses = ['red', 'yellow', 'green'] as const satisfies PatientStatusForTransport[];
```

For a bit more type safety down below where you assign a `PatientStatus` to a `PatientStatusForTransport`.

Reply:
Added in 544ddec5055dc291fdc6e659c29934b0b0aad536